Makefile.release

```diff
@@ -148,13 +143,25 @@ docker-push:
 ifeq ($(DOCKER),)
 	$(error "Please specify Docker registry to use. Use DOCKER=coredns for releases")
 else
-	@echo $(DOCKER_PASSWORD) | docker login -u $(DOCKER_LOGIN) --password-stdin
+	@# Experimental CLI is required for docker manifest to work
+	@# Pushes coredns/coredns-$arch:$version images
+	@# Creates manifest for multi-arch image
+	@# Pushes multi-arch image to coredns/coredns:$version
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	@echo $(DOCKER_PASSWORD) | $(DOCKER_CLI) login -u $(DOCKER_LOGIN) --password-stdin
 	@echo Pushing: $(VERSION) to $(DOCKER_IMAGE_NAME)
 	for arch in $(LINUX_ARCH); do \
-	    docker push $(DOCKER_IMAGE_NAME):coredns-$$arch ;\
+		$(DOCKER_CLI) push $(DOCKER_IMAGE_NAME)-$${arch}:$(VERSION) ;\
+		$(DOCKER_CLI) push $(DOCKER_IMAGE_NAME)-$${arch}:latest ;\
+	done
+	$(DOCKER_CLI) manifest create --amend $(DOCKER_IMAGE_NAME):$(VERSION) $(shell echo $(LINUX_ARCH) | sed -e "s~[^ ]*~$(DOCKER_IMAGE_NAME)\-&:$(VERSION)~g")
```

Comment:
can we pull the sed command out of this and assign to variable with a sensible name?

Reply:
Done!